gutenberg/acquire/metadata.py

```diff
@@ -201,6 +204,72 @@ def _check_can_be_instantiated(cls):
         del db
 
 
+class FusekiMetadataCache(MetadataCache):
+    _CACHE_URL_PREFIX = 'http://'
```

Comment:
What happens if their url is behind `https://` (as recommended in their docs for production servers)?

Reply:
See above: I was misled by some old docs. Added https to the white-list.